src/stores/useAccountStore/useAccountsStore.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { act, renderHook } from '@testing-library/react';
+
+import { useAccountStore } from './useAccountStore';
+
+describe('useAccountStore', () => {
+  it('should update mainContent correctly', () => {
+    const { result } = renderHook(() => useAccountStore());
+
+    const newContent = 'Test content';
+
+    act(() => {
+      result.current.updateMainContent(newContent);
+    });
+
+    expect(result.current.mainContent).toBe(newContent);
+  });
+
+  it('should update main content image correctly', () => {
```

Comment:
aqui a mesma coisa do teste acima, ainda sobre o should
```suggestion
  it('updates main content image', () => {
```